src/backend/GnuCashBudget.Adapter.Abstractions/Models/Expense.cs

```diff
@@ -0,0 +1,13 @@
+namespace GnuCashBudget.Adapter.Abstractions.Models;
+
+public class Expense
+{
+    public decimal Amount { get; set; }
+    public string Currency { get; set; }
+    public string? Category { get; set; }
+    public string? Description { get; set; }
+    public string Timestamp { get; set; }
+    public string? Merchant { get; set; }
+    public string? PaymentMethod { get; set; }
```

Comment:
enum?

Reply:
Changed in 736d262a624db46800a33e95484b24f4955ef831